search-sentiments-map/public/scripts/current-data-script.js

```diff
@@ -73,13 +69,23 @@ function setNewTrend(trend) {
       });
 }
 
+/** Bold and italicize the currently viewed trend. */
+function highlightCurrentTrend() {
+  let trendElements = document.getElementById('trends-list').childNodes;
+  trendElements.forEach(function(trendElement) {
+    trendElement.innerHTML = (trendElement.innerText === currentTopic) ?
+        '<span class="font-weight-bold font-italic">' + currentTopic +
+        '</span>' : trendElement.innerText;
```

Comment:
Use this formatting.

Then wrap that long line further if needed .

```suggestion
    trendElement.innerHTML = (trendElement.innerText === currentTopic)
        ? '<span class="font-weight-bold font-italic">' + currentTopic + '</span>'
        : trendElement.innerText;
```

Reply:
I'm changing it here, but we didn't use the formatting in other places. Do we want to change them all? For the record, we did: 
condition ?
        some long code when condition is true :
        some long code when condition is false;